index.js

```diff
@@ -272,7 +272,6 @@ Speech.prototype._escape = function (word) {
         word = word.replace(/>/g, '');
         word = word.replace(/"/g, '');
         word = word.replace(/'/g, '');
-        return word;
```

Comment:
I see you removed this return. This seems like a bug. Did you run the tests cases?

Reply:
my bad - I just updated and all tests are passing